core/client/src/mill/main/client/CodeGenConstants.java

```diff
@@ -20,19 +24,23 @@ public class CodeGenConstants {
   /**
    * The name of the root build file
    */
-  public static final String[] rootBuildFileNames = {"build.mill", "build.mill.scala", "build.sc"};
+  public static final List<String> rootBuildFileNames = Collections.unmodifiableList(Arrays.asList(
```

Comment:
We require Java 11 now right? 

So, we can use `List.of("build.mill", ...  ...)`?

Reply:
Yeah, makes sense. Thanks.